atd-vze/src/views/Crashes/UnitDetailsCard.js

```diff
@@ -35,29 +29,22 @@ const UnitDetailsCard = ({
 
   const { data: lookupSelectOptions } = useQuery(GET_UNIT_LOOKUPS);
 
-  if (loading) return "Loading...";
-  if (error) return `Error! ${error.message}`;
-
-  return loading ? (
-    <span>Loading...</span>
```

Comment:
i dont thinkkk we actually need this here

Reply:
also a few lines above it, it would return anyway "Loading...."

should that be removed too?

Reply:
Oh duh, now I see that you added those lines in this PR -- disregard my removal comment!!

Reply:
also took me a sec to see it is the lookup data loading state 🙃 Love how all of this works now!